pkg/core/native/native_neo.go

```diff
@@ -437,9 +451,8 @@ func (n *NEO) CalculateBonus(ic *interop.Context, value *big.Int, start, end uin
 	} else if value.Sign() < 0 {
 		return nil, errors.New("negative value")
 	}
-	si := ic.DAO.GetStorageItem(n.ContractID, []byte{prefixGASPerBlock})
-	var gr state.GASRecord
-	if err := gr.FromBytes(si.Value); err != nil {
+	gr, err := n.getSortedGASRecordFromDAO(ic.DAO)
```

Comment:
Why doesn't it use `n.gasPerBlock` here?

Reply:
I checked all users of `CalculateBonus`, so it actually can use cached values. Fixed.